src/Frontend/Modules/Profiles/Engine/Model.php

```diff
@@ -22,6 +23,21 @@ class Model
      */
     private static $avatars = [];
 
+    public static function maxDisplayNameChanges(): int
+    {
+        return FrontendModel::getModuleSetting('Profiles', 'max_display_name_changes', self::MAX_DISPLAY_NAME_CHANGES);
```

Comment:
@bjorvack could you maybe set this setting in the installer as well? that way it is clear when looking at the installer what settings are available?
You can keep the default here for websites that upgrade

Reply:
@bjorvack can you have a look at this?

Reply:
@carakas 6017f82